pixeltable/catalog/column.py

```diff
@@ -115,21 +115,30 @@ def __init__(
         # self.dependent_cols = set()  # cols with value_exprs that reference us; set by TableVersion
         self.id = col_id
         self.is_pk = is_pk
+        self.is_iterator_col = is_iterator_col
         self._media_validation = media_validation
         self.schema_version_add = schema_version_add
         self.schema_version_drop = schema_version_drop
 
-        self._stores_cellmd = stores_cellmd
+        if stores_cellmd is not None:
```

Comment:
Is there any way to get rid of the `stores_cellmd` override and make it a deterministic property of the other column attributes? (I find these kinds of overrides make the code more convoluted and invite bugs.)